README.md

```diff
@@ -47,21 +57,23 @@ git submodule init
 git submodule update
 ```
 
-### Using docker
+### Local deploy
+
+#### Using docker
 
 ```sh
 docker run -p 8080:8080 -d --rm --name devast --mount src=/path/to/syntax-tree-visualization,dst=/app,type=bind skril/devast:archlinux
 ```
 
 **Important:** change "path/to/syntax-tree-visualization" to the root directory of the project
 
-Alternatively on UNIX operating systems:
+on UNIX operating systems:
```

Comment:
It is important to state that this is an alternative way to run it, not continuation of previous one